stdlib/benchmarks/collections/bench_list.mojo

```diff
@@ -0,0 +1,88 @@
+# ===----------------------------------------------------------------------=== #
+# Copyright (c) 2024, Modular Inc. All rights reserved.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions:
+# https://llvm.org/LICENSE.txt
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ===----------------------------------------------------------------------=== #
+# RUN: %mojo-no-debug %s -t
+# NOTE: to test changes on the current branch using run-benchmarks.sh, remove
+# the -t flag. Remember to replace it again before pushing any code.
+
+from benchmark import Bench, BenchConfig, Bencher, BenchId, Unit, keep, run
+from memory import UnsafePointer
+from random import seed
+from collections import List
+from collections import Dict
+from time import now
+
+
+def benchmark_list_hint_trivial_type_int[length: Int, iterations: Int]() -> Int:
+    var start = now()
```

Comment:
**Suggestion** Instead of doing this stopwatch sort of thing, can we use the `benchmark` module as intended?  You can take a look at the other benchmarks for how this sort of thing can be done.  Thanks!